### PR TITLE
Add French version of run scripts tutorial

### PR DESCRIPTION
**Pull request by jadechoghari**
## What does this PR do?

This pull request adds a French translation of the run_scripts from English to French.

## Changes made

- Translated `run_scripts.md` from English to French.
- Updated the `_toctree.yml` to include the French version of the run_scripts tutorial.

## Why is this change important?

Providing documentation in multiple languages helps make the information accessible to a broader audience and supports users who are more comfortable reading in French. 🇫🇷
More docs to come. 🤗

## Review

Like last time I will tag:
- @amyeroberts (for the merge ✅)
- @younesbelkada (for the review 🇫🇷🎊)

### CODE REVIEW

**Comment by jadechoghari** [comment]
seems younes is off, will cc @ArthurZucker 🇫🇷 for this PR.

**Review by ebrisk-bit — APPROVED** [review]
Semble bien 🇫🇷🎊

**Comment by jadechoghari** [comment]
@NielsRogge, could you kindly confirm the French part? 
We just need one review from a maintainer with write access to close this small PR, been 5 days. 
🤗 Thank you! 🇫🇷

File: docs/source/fr/run_scripts_fr.md
Comment (ArthurZucker) [inline docs/source/fr/run_scripts_fr.md:22]:
```suggestion
Vous trouverez également des scripts que nous avons utilisé dans nos [projets de recherche](https://github.com/huggingface/transformers/tree/main/examples/research_projects) et des [exemples "legacy"](https://github.com/huggingface/transformers/tree/main/examples/legacy) qui sont des contributions de la communauté. Ces scripts ne sont pas activement maintenus et nécessitent une version spécifique de 🤗 Transformers qui sera probablement incompatible avec la dernière version de la librairie.
```

File: docs/source/fr/run_scripts_fr.md
Comment (ArthurZucker) [inline docs/source/fr/run_scripts_fr.md:32]:
```suggestion
Pour exécuter avec succès la dernière version des scripts d'exemple, vous devez **installer 🤗 Transformers à partir du code source** dans un nouvel environnement virtuel :
```

File: docs/source/fr/run_scripts_fr.md
Comment (ArthurZucker) [inline docs/source/fr/run_scripts_fr.md:81]:
```suggestion
Après avoir configuré la bonne version de la librairie, accédez au dossier d'exemple de votre choix et installez les prérequis spécifiques à l'exemple.
```

File: docs/source/fr/run_scripts_fr.md
Comment (ArthurZucker) [inline docs/source/fr/run_scripts_fr.md:92]:
```suggestion
Le script d'exemple télécharge et prétraite un jeu de données à partir de la bibliothèque 🤗 [Datasets](https://huggingface.co/docs/datasets/). Ensuite, le script affine un ensemble de données à l'aide de [Trainer](https://huggingface.co/docs/transformers/main_classes/trainer) sur une architecture qui prend en charge la tâche de résumé. L'exemple suivant montre comment ajuster le modèle [T5-small](https://huggingface.co/google-t5/t5-small) sur les données [CNN/DailyMail](https://huggingface.co/datasets/cnn_dailymail). Le modèle T5 nécessite un argument supplémentaire `source_prefix` en raison de la façon dont il a été entraîné. Cette invite permet à T5 de savoir qu'il s'agit d'une tâche de résumé.
```

File: docs/source/fr/run_scripts_fr.md
Comment (ArthurZucker) [inline docs/source/fr/run_scripts_fr.md:133]:
```suggestion
- Définissez le nombre de GPU à utiliser avec l'argument `nproc_per_node`.
```

File: docs/source/fr/run_scripts_fr.md
Comment (ArthurZucker) [inline docs/source/fr/run_scripts_fr.md:159]: unités de traitement de tenseur sounds super strange

File: docs/source/fr/run_scripts_fr.md
Comment (ArthurZucker) [inline docs/source/fr/run_scripts_fr.md:198]: entrainer et inférer

File: docs/source/fr/run_scripts_fr.md
Comment (ArthurZucker) [inline docs/source/fr/run_scripts_fr.md:236]:
```suggestion
Un exemple de script de résumé utilisant un ensemble de données personnalisé ressemblerait à ceci :
```
un script de résumé n'ap as de sense à revoir

**Review by ArthurZucker — COMMENTED** [review]
thanks for translating

File: docs/source/fr/run_scripts_fr.md
Comment (jadechoghari) [reply docs/source/fr/run_scripts_fr.md:159]: true, mais je pense que c'est correct:
https://phoenixnap.fr/glossaire/qu%27est-ce-qu%27une-unit%C3%A9-de-traitement-de-tenseur

File: docs/source/fr/run_scripts_fr.md
Comment (jadechoghari) [reply docs/source/fr/run_scripts_fr.md:198]: D'accord, vous pouvez la modifier
merci!

**Review by ArthurZucker — APPROVED** [review]

**Pull request merged**
